aptos-move/framework/src/module_metadata.rs

```diff
@@ -91,6 +99,24 @@ pub fn get_module_metadata(module: &CompiledModule) -> Option<RuntimeModuleMetad
     }
 }
 
+pub fn verify_metadata(metadata: &RuntimeModuleMetadataV1) -> bool {
+    for attrs in metadata.struct_attributes.values() {
```

Comment:
metadata. struct_attributes seems wrong?

Reply:
good catch... I was waiting for a ... this doesn't look to wild before moving forward... please review again..